demo/src/app.js

```diff
@@ -222,6 +223,74 @@ const validateUser = (user) => {
     );
 };
 
+const TableTabs = () => {
+    const [version, setVersion] = useState(0);
+
+    const columns = useMemo(
```

Comment:
Because it could (and in fact, at some moment, did) hide the failure in refreshing as columns were recreated at each render/call of TableTabs and such would trigger bigger changes in the virtual table that is to use it.